tests/unit/s2n_self_talk_broken_pipe_test.c

```diff
@@ -149,10 +149,10 @@ int main(int argc, char **argv)
         EXPECT_EQUAL(conn->actual_protocol_version, S2N_TLS12);
 
         /* Give client a chance to close pipe at the receiving end */
-        sleep(1);
+        sleep(4);
```

Comment:
the original sleep was introduced in this PR https://github.com/aws/s2n-tls/pull/3184